pkg/storaged/filesystem/mounting-dialog.jsx

```diff
@@ -319,6 +324,8 @@
                         opts.push("_netdev");
                     if (forced_options)
                         opts = opts.concat(forced_options);
+                    if (subvol)
+                        opts.push("subvol=" + subvol.pathname);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19690-20231205-210454-49836fdb-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)